compiler/rustc_lint/src/non_ascii_idents.rs

```diff
@@ -175,11 +175,8 @@ impl EarlyLintPass for NonAsciiIdents {
 
         // Sort by `Span` so that error messages make sense with respect to the
         // order of identifier locations in the code.
-        // We will soon sort, so the initial order does not matter.
-        #[allow(rustc::potential_query_instability)]
```

Comment:
This allow is ok, as there is an explaining comment with it. Or remove the sort?

Reply:
The vec is being sorted by spans, which seems to be deliberate judging from the comment.